src/app/donation-start/donation-start.component.ts

```diff
@@ -66,6 +66,8 @@ export class DonationStartComponent implements OnDestroy, OnInit {
   // but modified to make the separating space optional.
   private postcodeRegExp = new RegExp('^([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9]?[A-Za-z]))))\\s?[0-9][A-Za-z]{2})$');
 
+  private reservationMinutes = 15;
```

Comment:
@NoelLH worth making this an env variable?

Reply:
Yeah, good call @tbg-jasonfung. I'll make a new PR.